<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointROIWin</class>
 <widget class="QDialog" name="PointROIWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Zmin</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Xmin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Zmax</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ymin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ymax</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Xmax</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="YmaxlineEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="XmaxlineEdit"/>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="ZmaxlineEdit"/>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QPushButton" name="OKButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="XminlineEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="YminlineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="ZminlineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
